packages/providers/src/json-rpc-provider.ts

```diff
@@ -80,26 +81,37 @@ export class JsonRpcProvider extends Provider {
         return this.sendJsonRpc('status', []);
     }
 
+    /**
+     * Sends a signed transaction to the RPC
+     *
+     * @param signedTransaction The signed transaction being sent
+     * @param waitUntil
+     */
+    async sendTx(signedTransaction: SignedTransaction, waitUntil: TxExecutionStatus): Promise<TxOutcome> {
+        const bytes = encodeTransaction(signedTransaction);
+        return this.sendJsonRpc('send_tx', { signed_transaction: Buffer.from(bytes).toString('base64'), wait_until: waitUntil });
+    }
+
     /**
      * Sends a signed transaction to the RPC and waits until transaction is fully complete
      * @see [https://docs.near.org/docs/develop/front-end/rpc#send-transaction-await](https://docs.near.org/docs/develop/front-end/rpc#general-validator-status)
      *
      * @param signedTransaction The signed transaction being sent
      */
-    async sendTransaction(signedTransaction: SignedTransaction): Promise<FinalExecutionOutcome> {
+    async sendTransaction(signedTransaction: SignedTransaction): Promise<TxOutcome> {
         const bytes = encodeTransaction(signedTransaction);
-        return this.sendJsonRpc('broadcast_tx_commit', [Buffer.from(bytes).toString('base64')]);
+        return this.sendJsonRpc('broadcast_tx_commit', { signed_transaction: Buffer.from(bytes).toString('base64'), wait_until: 'Final' });
     }
 
     /**
      * Sends a signed transaction to the RPC and immediately returns transaction hash
      * See [docs for more info](https://docs.near.org/docs/develop/front-end/rpc#send-transaction-async)
      * @param signedTransaction The signed transaction being sent
-     * @returns {Promise<FinalExecutionOutcome>}
+     * @returns {Promise<TxOutcome>}
      */
-    async sendTransactionAsync(signedTransaction: SignedTransaction): Promise<FinalExecutionOutcome> {
+    async sendTransactionAsync(signedTransaction: SignedTransaction): Promise<TxOutcome> {
         const bytes = encodeTransaction(signedTransaction);
-        return this.sendJsonRpc('broadcast_tx_async', [Buffer.from(bytes).toString('base64')]);
+        return this.sendJsonRpc('broadcast_tx_async', { signed_transaction: Buffer.from(bytes).toString('base64'), wait_until: 'None' });
```

Comment:
From [here](https://github.com/near/nearcore/blob/447572bb775a5954ca7a5a4b65c85b45aab9e499/chain/jsonrpc/CHANGELOG.md#023): 
```
Added send_tx method which gives configurable execution guarantees options and potentially replaces existing broadcast_tx_async, broadcast_tx_commit
```
Won't it be better if these function just call `sendTx` with the proper `wait_until` parameter?